### DISCUSSION TITLE
504

### DISCUSSION THREAD

**Discussion by LSCube7** [spencerwooo/onedrive-vercel-index#294] [Q&A] [answered]
<img width="875" alt="image" src="https://user-images.githubusercontent.com/65765136/150287046-026a3889-0788-481d-8308-69b5646feee3.png">
<img width="976" alt="image" src="https://user-images.githubusercontent.com/65765136/150287100-d5872087-eb86-4ca9-9896-b5dfdc3a102e.png">
<img width="590" alt="image" src="https://user-images.githubusercontent.com/65765136/150287287-708872b0-140a-42ff-8e98-b84bc8c0959a.png">

请问这是什么问题？
访问超时？

**Comment by spencerwooo** [comment, marked as answer]
看 log 应该是没连到 Redis 上，但是你 REDIS_URL 看起来没什么大问题 🤔

可以本地 npm run dev 测试吗？我看你也修改了 client id 和 client secret，不过可能跟他俩没啥关系。

**Reply by LSCube7** [reply]
现在好了，不知道为什么要等一会才有访问权限🤔
